Files/Dialogs/SettingsDialog.xaml

```diff
@@ -15,19 +15,57 @@
         <ResourceDictionary>
             <x:Double x:Key="ContentDialogMaxWidth">800</x:Double>
             <Thickness x:Key="ContentDialogPadding">0</Thickness>
+            <Thickness x:Key="NavigationViewTopPaneHeight">40</Thickness>
+            <Thickness x:Key="TopNavigationViewTopNavGridMargin">10</Thickness>
             <SolidColorBrush x:Key="ContentDialogTopOverlay" Color="Transparent" />
             <ResourceDictionary.ThemeDictionaries>
                 <ResourceDictionary x:Key="Light">
                     <SolidColorBrush x:Key="NavigationViewExpandedPaneBackground" Color="Transparent" />
                     <SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="Transparent" />
+                    <SolidColorBrush x:Key="NavigationViewItemBackground" Color="{ThemeResource SolidBackgroundFillColorTertiary}" />
+                    <SolidColorBrush x:Key="TopNavigationViewItemBackgroundPointerOver"  Color="{ThemeResource SolidBackgroundFillColorTertiary}" />
+                    <SolidColorBrush x:Key="TopNavigationViewItemBackgroundPressed" Color="{ThemeResource SolidBackgroundFillColorTertiary}" />
+                    <SolidColorBrush x:Key="TopNavigationViewItemBackgroundSelected" Color="{ThemeResource SolidBackgroundFillColorTertiary}"/>
+                    <SolidColorBrush x:Key="TopNavigationViewItemBackgroundSelectedPointerOver" Color="{ThemeResource SolidBackgroundFillColorTertiary}"/>
+                    <SolidColorBrush x:Key="TopNavigationViewItemBackgroundSelectedPressed" Color="{ThemeResource SolidBackgroundFillColorTertiary}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrush" Color="{ThemeResource ControlStrokeColorDefault}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrushPointerOver" Color="{ThemeResource ControlStrokeColorDefault}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrushPressed" Color="{ThemeResource ControlStrokeColorDefault}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrushSelected" Color="{ThemeResource ControlStrokeColorDefault}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrushSelectedPointerOver" Color="{ThemeResource ControlStrokeColorDefault}"/>
+                    <SolidColorBrush x:Key="NavigationViewItemBorderBrushSelectedPressed" Color="{ThemeResource ControlStrokeColorDefault}"/>
```

Comment:
@chingucoding the border brush doesn't seem to be working, is there a different resource I need to change as well?

Reply:
The V2 NavigationViewItem presenter style doesn't have border setters like the V1 version has (and neither does the NavigationViewItem itsself). I think this is worth opening an issue on the WinUI repo since you cannot effectively deal with a border than. Would you like to do that or should I?

Reply:
@chingucoding https://github.com/microsoft/microsoft-ui-xaml/issues/6283
Is there a way to use v1 in the meantime?

Reply:
Not really without having to use v1 styles with larger parts of the app. You could use a custom NavigationViewItemPresenter style to add the border back though. But again, retemplating is not that great as an option.